h/services/search_index/_queue.py

```diff
@@ -211,13 +211,16 @@ def _get_jobs_from_queue(self, limit):
             .all()
         )
 
-    def _job_query(self, tags=None):
+    def _job_query(self, tags=None, hide_scheduled=True):
         now = datetime.utcnow()
 
         query = self._db.query(Job).filter(
-            Job.name == "sync_annotation", Job.scheduled_at < now, Job.expires_at >= now
+            Job.name == "sync_annotation", Job.expires_at >= now
         )
 
+        if hide_scheduled:
+            query = query.filter(Job.scheduled_at < now)
+
```

Comment:
Seems pretty straight forward. I like how this auto-documents what this filter is doing as well.